src/talk/renderer/Viewer/ViewerHandlerImages.vue

```diff
@@ -14,40 +15,17 @@ const props = defineProps({
 	},
 })
 
-const src = computed(() => {
-	if (!props.file) {
-		return null
-	}
-
-	const searchParams = new URLSearchParams(Object.entries({
-		fileId: props.file.fileid,
-		x: Math.floor(screen.width * devicePixelRatio),
-		y: Math.floor(screen.height * devicePixelRatio),
-		a: 'true',
-		etag: props.file.etag,
-	})).toString()
-
-	return generateUrl(`/core/preview?${searchParams}`)
-})
+const src = computed(() => generateFilePreviewUrl(props.file.fileid, props.file.etag))
```

Comment:
Is it always guaranteed to have `.fileid` and `.etag` ?

Reply:
It should, from here: https://github.com/nextcloud/spreed/blob/52082e0caf61dfbb52f67e87e3a46735428b8078/src/composables/useViewer.js#L112